catalogs/catalog-hadoop/build.gradle.kts

```diff
@@ -50,6 +53,7 @@ dependencies {
 
   testImplementation(libs.minikdc)
   testImplementation(libs.hadoop3.minicluster)
+  testImplementation("org.apache.hadoop:hadoop-aws:$hadoopTestVersion")
```

Comment:
@yuqi1129 Do you have some suggestion? It's bad to use `common.lang`, because we should use `common.lang3`. But it seems a little weird to use hadoop 3.3.6 directly.

Reply:
I have no better idea about it as the version of `MiniKDC` should be `3.3.6`,  There exist some compatible issues if we download it to `3.1.0` I remember,  Is it a good solution to update the Hadoop version to `3.3.6`?

Reply:
I prefer upgrading the hadoop version. It's hard to solve legacy version compatible issues.

Reply:
@jerryshao What's your opinion on this? If no other good ideas, I'm open to upgrading the Hadoop version.

Reply:
I think we'd better upgrading to a newer version.